<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PresentationClient</class>
 <widget class="QMainWindow" name="PresentationClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PresentationClient</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="connectionButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>531</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>761</width>
      <height>501</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="showAPresentationButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>531</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Show a presentation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="downloadVoiceRecordButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>531</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Download voice record</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
